### DISCUSSION TITLE
Support validateInputOnBlur

### DISCUSSION THREAD

**Discussion by gdostie** [mantinedev/mantine#2291] [Feature requests]
`useForm` hook from `@mantine/form` already supports `validateInputOnChange` option. From a user standpoint, showing errors on change can seem a bit too agressive. Showing the errors on blur is usually a friendlier behaviour while remaining more proactive than validating on submit.

Example of usage:
All fields
```js
const form = useForm({ validateInputOnBlur: true });
```
Some fields
```js
const form = useForm({ validateInputOnBlur: ['name', 'email'] });
```

**Comment by rtivital** [comment]
Hi @gdostie, thanks for a feature request, it seems useful, so I'll add it in one of the next minor versions
